verl/single_controller/ray/base.py

```diff
@@ -431,13 +436,13 @@ def _init_with_detached_workers(self, worker_names, worker_handles):
         self._workers = workers
         self._world_size = len(worker_names)
 
-    def _get_master_addr_port(self, pg):
+    def _get_master_addr_port(self, pg, bundle_index):
```

Comment:
Maybe replace `bundle_index` with another name like `worker_idx` because `bundle` belongs to the concept of placement group (resources allocation) while getting master addr & port is part of creating workers.